### PR TITLE
Use format to parts

### PR DESCRIPTION
**Pull request by klzns**
#### What problem is this solving?

Fix issue of formatting decimal places like: #11 

#### How should this be manually tested?

https://breno--storecomponents.myvtex.com/
https://breno--vtexromania.myvtex.com/

#### Checklist/Reminders

- [x] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [X] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Screenshots or example usage

image

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by vitorflg — APPROVED** [review]
i think it's not the case here but if we need to customize that for other countries, would be nice to separate that logic in specific files!

**Review by kaisermann — APPROVED** [review]

**Review by iaronaraujo — APPROVED** [review]

File: react/FormattedCurrency.tsx [hunk 1]
```diff
@@ -1,21 +1,102 @@
...
+})
+
+const hasFormatToParts = 'formatToParts' in formatter
+
+const CSS_HANDLES = [
+  'currencyCode',
+  'currencyLiteral',
+  'currencyInteger',
+  'currencyGroup',
+  'currencyDecimal',
```
Comment (victorhmp) [inline react/FormattedCurrency.tsx:20]: How about `currencyDecimalSeparator`?
Then `currencyFraction` could become `currencyDecimal`

File: react/FormattedCurrency.tsx [hunk 2]
```diff
@@ -1,21 +1,102 @@
...
+
+  if (culture.customCurrencyDecimalDigits != null) {
+    formatOptions.minimumFractionDigits = culture.customCurrencyDecimalDigits
+  }
+
+  /**
+   * The default Romanian currency format is wrong
+   * https://stackoverflow.com/questions/57526989/return-correct-currency-for-intl-numberformat-romanian-lei
+   */
+  const isRomanian =
```
Comment (victorhmp) [inline react/FormattedCurrency.tsx:71]: Wow! Nice catch 😅

**Review by victorhmp — APPROVED** [review]

File: react/FormattedCurrency.tsx (hunk 1 above)
Comment (klzns) [reply react/FormattedCurrency.tsx:20]: @victorhmp I prefer that way it is right now because I'm using the same names the Native JS API uses

Comment (victorhmp) [reply react/FormattedCurrency.tsx:20]: Oooh! Did not know that. Awesome then 😄

**Comment by klzns** [comment]
I udpated the docs and added a span around

**Review by kaisermann — APPROVED** [review]

**Review by guerreirobeatriz — APPROVED** [review]

**Review by icazevedo — APPROVED** [review]

**Pull request merged**